src/.nuget/packageLoad.targets

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" InitialTargets="_RestoreBuildTools" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    Contains targets to do NuGet packages. The following tasks need to be performed:
+        Parameters:
+          NuGetConfigDir         - this directory may contain the following files:
+                                   NuGet.Config - configuration file
+                                   packages.config - old-facioned way to installed packages
+                                   project.json - new way to install packages
+          NuGetDir               - location to install packages
+          SolutionFile           - if set, pass to NuGet.exe to restore everything for the solution
+          SolutionRoot           - if SolutionFile is not set and this is set, process all project.json
+                                   files under this directory. Excludes project.json in NuGetConfigDir.
+          InstallationDependency - list of files that, when changed, need to cause reinstall
+          BuildToolsSemaphore    - Created when build tools are restored
+        1. Downloads NuGet.exe
+        2. Handle packages.config and download packages it specifies
```

Comment:
Can you be more concrete than "handle"?  This note *almost* tells me what I need to know but I still have to read the targets.